src/Symfony/Component/Messenger/Transport/Sync/SyncTransport.php

```diff
@@ -13,18 +13,27 @@
 
 use Symfony\Component\Messenger\Envelope;
 use Symfony\Component\Messenger\Exception\InvalidArgumentException;
-use Symfony\Component\Messenger\Stamp\ForceCallHandlersStamp;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Symfony\Component\Messenger\Stamp\ReceivedStamp;
+use Symfony\Component\Messenger\Stamp\SentStamp;
 use Symfony\Component\Messenger\Transport\TransportInterface;
 
 /**
- * A "fake" transport that marks messages to be handled immediately.
+ * Fake transport that immediately marks messages as received and dispatches for handling.
```

Comment:
In fact it’s not a “fake”, but just some kind of transport.
Should we remove this kind of wording?